packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,36 +1,45 @@
 {
-  "finalSystemOwner": "DUMMY",
-  "controller": "DUMMY",
+  "numDeployConfirmations": 1,
+
+  "finalSystemOwner": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+  "controller": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+
+  "l1StartingBlockTag": "FILL_IN_DAY_OF",
 
-  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
   "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1200,
+  "maxSequencerDrift": 600,
   "sequencerWindowSize": 3600,
-  "channelTimeout": 120,
+  "channelTimeout": 300,
 
-  "p2pSequencerAddress": "DUMMY",
+  "p2pSequencerAddress": "0x715b7219D986641DF9eFd9C7Ef01218D528e19ec",
   "batchInboxAddress": "0xff00000000000000000000000000000000000420",
-  "batchSenderAddress": "DUMMY",
+  "batchSenderAddress": "0x7431310e026B69BFC676C0013E12A1A11411EEc9",
+
   "l2OutputOracleSubmissionInterval": 20,
```

Comment:
Josh/me suggested 20 min, but it's in number of blocks, so that would be `20*60/12=100` Are we intentionally setting it to 20?

Reply:
As of now yes - cc @K-Ho

Reply:
So right now on Optimism Goerli, we submit state batches roughly once every 5 minutes https://goerli.etherscan.io/address/0x9c945aC97Baf48cB784AbBB61399beB71aF7A378. This roughly costs 10 Goerli ETH per month. 
If we choose 20 blocks here, we will replicate the devex of testing withdrawals on Optimism Goerli today, with a little bit more waiting due to two step withdrawals - every time a developer wishes to test sending a withdrawal, they will have to wait 0-5 minutes before proving their withdrawal and then 10 seconds before finalizing their withdrawal.

I'd recommend we parameterize this value to as low of a value as we think the protocol can handle - if we can safely set it to **2 blocks**, that would be a much better devex for testing withdrawals (the average time to try a withdrawal would go from ~3 minutes down to <1 minute) for just a little bit of extra Goerli ETH spend on our end. 

We've had developers complain in the past about it being slow to test if they are doing withdrawals correctly given the time required to test a single end-to-end withdrawal.

Reply:
FYI I'm pretty sure that this parameter is the number of **L2 Blocks**. 20 L2 blocks -> submit every 40s which is a pretty fast UX. We can drop it down to every 24s or 30s (i.e. ~2 L1 blocks). This will cost us ~100 Goerli ETH per month which seems fine.